README.md

```diff
@@ -105,6 +105,7 @@ Metadata heuristics:
 | release_zero | Identify packages with an release version that's 0.0 or 0.0.0 |
 | typosquatting | Identify packages that are named closely to an highly popular package |
 | potentially_compromised_email_domain | Identify when a package maintainer e-mail domain (and therefore package manager account) might have been compromised |
+| unclaimed_maintainer_email_domain | Identify when a package maintainer e-mail domain (and therefore package manager account) is currently unclaimed and could be registered by an attacker |
```

Comment:
```suggestion
| unclaimed_maintainer_email_domain | Identify when a package maintainer e-mail domain (and therefore npm account) is unclaimed and can be registered by an attacker |
```

Reply:
fixed in https://github.com/DataDog/guarddog/pull/346/commits/d10a646a725f3ccb6d66a5c9e5dce7326e1b542d